src/utils/helper_functions.ts

```diff
@@ -2,10 +2,83 @@
  *This file contains helper functions, called across various componenets
  */
 
-export const tempFunction = () => {
-  let testNum = 0;
+import dataSet from "../datasets/taxon_data.json";
+import { classificationLevelArray } from "./constants";
+import { taxonInterface, taxonInterfaceArray } from "./datagrab";
+
+/*
+ * Accepts a single lk_taxon element, which is
+ * an array containing the current taxon levels and values
+ * returns the current taxon_id value.
+ */
+export const getCurrentTaxonID = (currTaxonArray: taxonInterface): string => {
```

Comment:
does this work?

Reply:
haven't updated +  used it yet. so not 100% sure if it works yet.